SubmittingPatches.rst

```diff
@@ -519,3 +513,134 @@ decreasing the likelihood of your MIME-attached change being accepted.
 Exception: If your mailer is mangling patches then someone may ask
 you to re-send them using MIME.
 
+
+BACKPORTING
+===========
+
+Ceph has a `Stable Releases and Backports`_ team, staffed by volunteers,
+which is charged with maintaining the stable releases and backporting bugfixes
+from the master branch to them.
+
+Even if your objective is to get a bugfix into a stable release, your initial
+PR still needs to target master.
+
+.. _`Stable Releases and Backports`: http://tracker.ceph.com/projects/ceph-releases/wiki
+
```

Comment:
worth mentioning that this wiki also describes various workflows in the backporting lifecycle

Reply:
Done